src/components/questionStructure/Task/variants/HandwritingTask/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { Pencil2Icon } from '@radix-ui/react-icons'
+import { Button, Card, Dialog, Flex, Text } from '@radix-ui/themes'
+import { MathJax } from 'better-react-mathjax'
+import { FC } from 'react'
+import { useParams } from 'react-router-dom'
+
+import { DEFAULT_TEST_USERNAME } from '../../../../../utils/globalConstants'
+import { TaskType } from '../../constants'
+import { TaskBaseProps } from '../../types'
+import HandwritingEditor from './HandwritingEditor'
+import { ViewOnlyCanvas } from './ViewOnlyCanvas'
+import './index.scss'
+
+export interface HandwritingTaskProps extends TaskBaseProps<string> {
+  type: TaskType.PROCESSED_HANDWRITING
+  questionText: string
+}
+
+export const HandwritingTask: FC<HandwritingTaskProps> = ({
+  answer,
+  onAnswerUpdate,
+  disabled = false,
+  questionText,
+}) => {
+  const { username = DEFAULT_TEST_USERNAME } = useParams()
+
+  return (
+    <Dialog.Root>
+      <Dialog.Trigger>
+        <ViewOnlyCanvas initialData={{}} />
+      </Dialog.Trigger>
+      <Flex gap="3" align="center">
+        <Card className="latex-preview">
+          <Flex gap="3" p="3" align="center" justify="between">
+            <MathJax style={{ flex: '1 0 0', textAlign: 'center' }}>
+              <Text>{answer ? `\\( ${answer} \\)` : 'No Answer'}</Text>
+            </MathJax>
+          </Flex>
+        </Card>
+        <Dialog.Trigger>
+          <Button variant="surface" disabled={disabled} style={{ cursor: 'pointer' }} size="4">
+            Edit answer <Pencil2Icon width="1.5rem" height="1.5rem" />
+          </Button>
+        </Dialog.Trigger>
+      </Flex>
+
+      <Dialog.Content style={{ minWidth: '90vw', height: '80vh' }}>
```

Comment:
Style in css file with suitable class name

Reply:
Done in 445b858